samples/04.0-file-transfer/integration-tests/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     testImplementation(testFixtures(project(":common:util")))
     testImplementation(testFixtures(project(":launchers:junit")))
+    testImplementation(project(":samples:04.0-file-transfer:client"))
```

Comment:
Seems a bit weird to have a dependency with a main class in it. Consider taking main class into separate module.

Reply:
Moved